hack/build/config.sh

```diff
@@ -49,7 +49,7 @@ VERBOSITY=${VERBOSITY:-1}
 PULL_POLICY=${PULL_POLICY:-IfNotPresent}
 NAMESPACE=${NAMESPACE:-cdi}
 
-KUBERNETES_IMAGE="k8s-1.11.0@sha256:e02ec414c7673a3644b5ba742b550a124c9195eaacb280151406bf9e8201a95f"
+KUBERNETES_IMAGE="k8s-multus-1.13.3"
```

Comment:
We need multus?  Or 1.13?  Or both?

Reply:
The only available 1.13 image is the multus one. Not sure why.

Reply:
I assume that was the case before https://github.com/kubevirt/containerized-data-importer/pull/724?

Reply:
Yeah, you are right, they added a non multus one in kubevirtci at this point, so once rebased on master that should be correct now.

Reply:
Done.